### PR TITLE
first run through of python package rewrite

### PR DESCRIPTION
**Pull request by ChristinaLK**
When testing, don't use /public/osg as the home of the python tar.gz files, but use /public/ckoch5.

### CODE REVIEW

File: examples/manage-python-packages.md [hunk 1]
```diff
@@ -1,175 +1,185 @@
...
+	
+		$ tar -xzf python37.tar.gz
+		$ export PATH=$PWD/python/bin:$PATH
 
-     $ module load python/3.7.0
+	To test that the installation worked, run the following command: 
+	
+		$ which python3
+	
+	The output line should start with your home directory.
```
Comment (rachellombardi) [inline examples/manage-python-packages.md:59]:
```suggestion
	The output line should start with your home directory or with a tilde (~) indicating the path of your home directory.
```

File: examples/manage-python-packages.md [hunk 2]
```diff
@@ -1,175 +1,185 @@
...
 
-Install each package that you need for your job using the `pip install` command.  Once 
-you are done, you can leave the virtual environment: 
+In order to run Python in a job, you will need to create a script (the job's "executable") 
+that unzips the OSG-provided copy of Python and your own Python packages, then 
+executes your Python script. 
 
-    (my_env)$ deactivate
+A sample script appears below. After the first line, the lines starting
+with hash marks are comments . You should replace \"myscript.py\" with
```
Comment (rachellombardi) [inline examples/manage-python-packages.md:115]:
```suggestion
with hash marks are comments. You should replace \"myscript.py\" with
```

**Review by rachellombardi — APPROVED** [review]
This looks good - it is very clear and detailed. I was able to complete all steps in both guides successfully. The only items that I ran into not being able to get working were the various embedded links to our other guides (even guides that have been published already). If that is normal behavior, then I think it's ready to be pushed to the website with just a few minor changes.

File: examples/build-python.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+	Once this tar.gz file is created, move it to your public directory: 
+
+		$ mv python.tar.gz /public/username
+
+7. **Submit jobs**
+
+	From here, follow the instructions in our [Run Python Scripts on the OS Pool][python-pkgs] 
+	guide, replacing the `stash:///osgconnect/public/osg` links with links to the copy of 
+	Python in your own `/public` directory. 
+
```
[the remark this answers is not in the document]
Comment (ChristinaLK) [reply examples/build-python.md:108]: this is a good reminder - i'll be adding the link to the python guide once i'm ready to merge.

**Comment by ChristinaLK** [comment]
We're going to go with containers instead of tarballs.

**Pull request closed without merging**